model/publishing/delivery/tasks/RemoteTaskStatusSynchroniser.php

```diff
@@ -183,11 +190,11 @@ private function callRemoteApi(string $url): array
      * @param string $deliveryUri
      * @param string $testUri
      */
-    private function triggerRemoteDeliveryCreatedEvent(?string $remoteDeliveryId, string $deliveryUri, string $testUri): void
+    private function triggerRemoteDeliveryCreatedEvent(?string $remoteDeliveryId, string $deliveryUri, string $testUri, ?string $alias): void
```

Comment:
`?string`
is php 7.4 version and i am bit reluctant that we have updated all environments to that version 
i would suggest to you to use the safe approach

Reply:
that's not true,  verified again , good job